xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/cssVariablesInit.css

```diff
@@ -0,0 +1,309 @@
+## Those should all be defined in variables.less and variablesInit.vm
+## Some values are changed from the bootstrap default in order to make sure there's no accessibility contrast issue
+## The default for these CSS values is directly taken from variablesInit.vm
+## The key is their name and the value is the string representing the value in CSS syntax.
+#set($xsVariables = {
+"figure-caption-font-size": "90%",
+"figure-caption-color": "var(--text-muted)",
+"loginFormMaxWidth": "450px",
+"login-form-max-width": "450px",
+"target-size-minimum": "24px",
+"target-size-recommended": "44px",
+"border-width": "1px",
+"headings-font-weight": "400",
+"nav-tabs-active-link-hover-bg": "var(--xwiki-page-content-bg)",
+"border-radius-base": "7px",
+"border-radius-large": "10px",
+"border-radius-small": "5px",
+"button-spacing": "2px",
+"button-border-width": "1px",
+"edit-section-relative-height": ".7",
+"main-padding": "calc(var(--grid-gutter-width) * 2)",
+"font-size-headings-scale": "1.190",
+"font-size-headings-scale-min": "1.125",
+"min-screen-size": "768",
+"max-screen-size": "1200",
+"min-h1-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale-min), 4))",
+"max-h1-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale), 4))",
+"min-h2-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale-min), 3))",
+"max-h2-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale), 3))",
+"min-h3-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale-min), 2))",
+"max-h3-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale), 2))",
+"min-h4-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale-min), 1))",
+"max-h4-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale), 1))", 
+"min-h6-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale-min), -1))",
+"max-h6-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale), -1))",
+"min-document-title-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale-min), 5))",
+"max-document-title-font-size": "calc(var(--font-size-base) * pow(var(--font-size-headings-scale), 5))",
+"100vw": "100vw",
+"int-viewport-width": "calc(10000 * tan(atan2(var(--100vw), 10000px)))",
+"font-size-document-title": "calc(max(var(--min-document-title-font-size) * 1px, min(var(--max-document-title-font-size) * 1px, var(--min-document-title-font-size) * 1px + (var(--max-document-title-font-size) - var(--min-document-title-font-size)) * (var(--int-viewport-width) - var(--min-screen-size)) / (var(--max-screen-size) - var(--min-screen-size)))))",
+"font-size-h1": "calc(max(var(--min-h1-font-size), min(var(--max-h1-font-size), var(--min-h1-font-size) + (var(--max-h1-font-size) - var(--min-h1-font-size)) * (var(--int-viewport-width) - var(--min-screen-size)) / (var(--max-screen-size) - var(--min-screen-size)))))",
```

Comment:
I'm not fan of seeing https://github.com/xwiki/xwiki-platform/blob/92c46a63a954990fb64e95b78a4a4e112cb1583a/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/type.less#L32 being duplicated.